data/core/help.cfg

```diff
@@ -390,7 +390,61 @@ If a strike is determined to hit, it will always do at least 1 point of damage.
+
+" +
+            #po: The help page has a set of images above this paragraph, in the same order as the bullet points
+            _ "• Most units have a circular ellipse.
+• Units that can recruit have a seven-pointed star.
+• <ref>dst='crowns' text='Hero units'</ref> have a smaller variant of the seven-pointed star, these units can’t recruit.
+• Units without a <ref>dst='movement' text='zone of control'</ref> have a broken version of the shape.
+• Some campaigns start with a level zero leader. Able to recruit but lacking a ZoC, these units have a broken version of the seven-pointed star."
+    [/topic]
+    # wmllint: markcheck on
+
+    # wmllint: markcheck off
+    [topic]
+        id=crowns
+        title= _ "Crowns and loyal markers"
+        text=_ "Some units have an additional marker superimposed on their <ref>dst='orbs' text='orb'</ref>, and a corresponding shape to their <ref>dst='orbs' text='ellipses'</ref>:" + "
+
+" +
+            "<img>src='help/empty-square.png~CROP(0,0,72,36)~BLIT(misc/leader-crown.png~CROP(4,0,36,18)~XBRZ(2))'</img>" +
+            "<img>src='help/empty-square.png~CROP(0,0,72,36)~BLIT(misc/hero-icon.png~CROP(4,0,36,18)~XBRZ(2))'</img>" +
+            "<img>src='help/empty-square.png~CROP(0,0,72,36)~BLIT(misc/leader-expendable.png~CROP(4,0,36,18)~XBRZ(2))'</img>" +
+            "<img>src='help/empty-square.png~CROP(0,0,72,36)~BLIT(misc/loyal-icon.png~CROP(4,0,36,18)~XBRZ(2))'</img>" + "
+" +
+            "<img>src='help/empty-square.png~BLIT(misc/ellipse-leader-top.png)~BLIT(misc/ellipse-leader-bottom.png)~BLIT(units/orcs/leader.png~TC(1, magenta))~BLIT(misc/leader-crown.png)'</img>" +
+            "<img>src='help/empty-square.png~BLIT(misc/ellipse-hero-top.png)~BLIT(misc/ellipse-hero-bottom.png)~BLIT(units/dunefolk/herbalist/alchemist.png~TC(1, magenta))~BLIT(misc/hero-icon.png)'</img>" +
+            "<img>src='help/empty-square.png~BLIT(misc/ellipse-leader-top.png)~BLIT(misc/ellipse-leader-bottom.png)~BLIT(units/human-magi/white-mage.png~TC(1, magenta))~BLIT(misc/leader-expendable.png)'</img>" +
+            "<img>src='help/empty-square.png~BLIT(misc/ellipse-top.png)~BLIT(misc/ellipse-bottom.png)~BLIT(units/human-loyalists/spearman.png~TC(1, magenta))~BLIT(misc/loyal-icon.png)'</img>" + "
+
+" +
+            #po: The help page has a set of images above this paragraph, in the same order as the first set of bullet points
+            _ "• Leaders are shown with a <bold>text='gold crown'</bold>.
+• Heroes have a <bold>text='blueish-silver crown'</bold>.
+• Expendable leaders are shown with a <bold>text='silver crown'</bold>.
+• <ref>dst='traits_loyal' text='Loyal units'</ref> have the <bold>text='loyal icon'</bold>.
+
+Of these categories, only leaders are built into the basic game rules and game engine. The others are found in campaigns, and are characters that form part of the narration. The exact rules for them are set by their campaign, which might deviate from these standard expectations about them:
```

Comment:
The "and game engine" should be removed. It's not useful information to the large majority of people who'd be reading this help entry.

Reply:
It also kinda sounds redundant - the game engine is the implementation of the game rules, so it's almost like you're saying the same thing twice?

Reply:
The information that I want to convey here is that units with canrecruit=yes alway have this exact ellipse (or its noZoC variant), and AFAIK WML can't override that with a different shape.

Reply:
The point being though that people who would want to know that information won't be reading the in-game help page to find it, they'd look on the wiki.